egasub/cli.py

```diff
@@ -49,8 +49,12 @@ def report(ctx, source):
 
 
 @main.command()
+@click.option('--ega_submitter_account', prompt=True)
+@click.option('--ega_submitter_password', prompt=True,hide_input=True)
+@click.option('--icgc_id_service_token', prompt=True)
+@click.option('--icgc_project_code', prompt=True)
```

Comment:
This may not fit our situation, before collecting this information, the code needs to check whether it is already inside a 'workspace'. If yes, then there is no need to collect this.